### DISCUSSION TITLE
Power BI DAX Suggestion 📊🔥

### DISCUSSION THREAD

**Discussion by theoyinbooke** [microsoft/30daysof#196] [Show and tell]
![project and feedback](https://github.com/microsoft/30daysof/assets/13628137/dee68006-d5f2-413e-9af7-a861f854009f)

#Your Project and Feedback 
Thank you for participating. We hope you enjoyed using the DAX Suggestion Feature while working on the challenge.

It's time for you to share with us what you worked on and tell us where the feature couldn't help address.

**Comment by theoyinbooke** [comment]
Can't wait to see you all submit your projects

**Reply by omotox77** [reply]
Hello ,

Please link to my Product on the Production Performance[](https://github.com/omotox77/Project1/blob/main/README.md)

**Comment by Solozys3** [comment]
How do I submit the project? I am done with it. Please find the link attached here.

https://github.com/Solozys3/AdventureWorks2019

**Comment by Adesoyin** [comment]
Good day please, here is the link to my submission 

https://github.com/Adesoyin/Product_Production-Project/blob/main/README.md

**Comment by Tolu07** [comment]
my submission ![](here) https://github.com/Tolu07/Inventory-Analysis

**Comment by iamcbn** [comment]
Hello!

Here is my project. I would definitely like to hear your genuine feedback
https://github.com/iamcbn/Operational-dashboard-Power-BI-Saturday

**Comment by DebbyNiola** [comment]
Hello, my Adventure works trial
https://github.com/DebbyNiola/AdventureWorks-2019-Production-

**Comment by Pelumi-Analyst** [comment]
Hello boss @theoyinbooke 
Kindly check my submission through this link - https://github.com/Pelumi-Analyst/Production-Performance-in-Manufacturing-Industry/tree/main

Thanks as I await your comment on it.

**Comment by Amarabright** [comment]
https://github.com/Amarabright/Production-Performance

I guess I took too long for this. However, it was a great learning experience.
Thank you for this.
